modules/ingestor/src/graph/sbom/cyclonedx.rs

```diff
@@ -192,6 +192,17 @@ impl SbomContext {
             for right in left.depends_on.iter().flatten() {
                 creator.relate(right.clone(), Relationship::DependencyOf, left.ref_.clone());
             }
+
+            // https://github.com/trustification/trustify/issues/1131
+            // Do we need to qualify this so that only "arch=src" refs
+            // get the GeneratedFrom relationship?
```

Comment:
good question - originally I thought we should think of 'red hat' mode as only applying to ingestion rules eg. we do not have to bake anything into rest api ... or add any additional metadata because we know a src vs binary package already.